models/istio_validation.go

```diff
@@ -175,6 +175,10 @@ var checkDescriptors = map[string]IstioCheck{
 		Message:  "KIA0004 No matching workload found for the selector in this namespace",
 		Severity: WarningSeverity,
 	},
+	"generic.exportto.namespacenotfound": {
```

Comment:
Small comment here: team decided to sort this associative map by its keys. I'd move this validation as the first of the selector block.
I know that then the validation references aren't sorted. It is weird. However, I've open this [issue to fix that](https://github.com/kiali/kiali/issues/4173). The duplication of keys are pretty unnecessary and tend to errors and more work when troubleshooting, developing, etc.

Reply:
can we leave this for later?

Reply:
I'd say to move this one sorted alphabetically by the hash id. Just not to break the current sorting. If the mentioned issue makes it to the backlog, we'll re-arrange everything. But I'd rather keep this consistently sorted. Not to left it broken. thx.

Reply:
ah, I see now, this is not sorted correctly